src/main/java/com/unosquare/admin_core/back_end/viewModels/holidays/HolidayViewModel.java

```diff
@@ -11,20 +11,12 @@
 public class HolidayViewModel {
 
         private int eventId;
-
         private LocalDate startDate;
-
         private LocalDate endDate;
-
         private EmployeeViewModel employee;
-
         private EventStatus eventStatus;
-
         private EventType eventType;
-
         private boolean isHalfDay;
-
         private LocalDate lastModified;
-
```

Comment:
Quick question as I've seen quite a lot of this in this PR, do you have some sort of formatter that's going through and stripping out new lines or something?